apex-app-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -50,6 +50,12 @@
       </includes>
     </fileSet>
     <fileSet filtered="true" encoding="UTF-8">
+      <directory>src/main/resources</directory>
+      <includes>
+        <include>**/apexlog4j.properties</include>
```

Comment:
We are having a custom behavior for logging, which doesn't follow standard log appender formats. I feel that's a reason I want to use different cofig name and make it clear to users, this is different than standard log4j appenders.